### PR TITLE
Update eh to v1.0.0

### PR DESCRIPTION
**Pull request by Sycrosity**
+ updated `Cargo.toml`'s
+ replace `DelayUs` with `DelayNs`
+ add various minor reference changes

### CODE REVIEW

**Review by avsaase — APPROVED** [review]

**Comment by Sycrosity** [comment]
Is there anything now blocking this from merging?

**Comment by avsaase** [comment]
I don't see any activity from the maintainer in this repo since January last year. 

@VersBinarii can you say something about your plans for maintaining this crate? If you don't have the time for it (totally fine of course) then it I think it makes sense for someone to do a fork.

**Comment by Sycrosity** [comment]
> @VersBinarii can you say something about your plans for maintaining this crate? If you don't have the time for it (totally fine of course) then it I think it makes sense for someone to do a fork.

I would be quite happy with forking and maintaining this project, if they do not have the time - considering I am using this crate and board extensively at current.

**Review by VersBinarii — APPROVED** [review]

**Pull request merged**
